gulpfile.js/tasks/clean.js

```diff
@@ -3,7 +3,7 @@ var del    = require('del')
 var config = require('../config')
 
 var cleanTask = function (cb) {
-  del([config.root.dest]).then(function (paths) {
+  del([config.root.dest + '/**', '!' + config.root.dest]).then(function (paths) {
```

Comment:
ah, nice. Is the `'!' + config.root.dest` necessary? Also, not sure, if `del` normalizes paths automatically (probably), but might be worth using `path.join(config.root.dest, '/**')` to normalize the path across platforms ( like windows).